### PR TITLE
Readme changes

### PR DESCRIPTION
**Pull request by marellapudihimamsu**
please review

### CODE REVIEW

**Review by AdityaKonduri79 — COMMENTED** [review]
Test

**Review by AdityaKonduri79 — CHANGES_REQUESTED** [review]
Test

File: newreactproj/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-This is a calculator app
+This is a calculator app.
```
Comment (AdityaKonduri79) [inline newreactproj/README.md:1]: Test

Comment (marellapudihimamsu) [reply newreactproj/README.md:1]: Test

**Review by AdityaKonduri79 — APPROVED** [review]
ok

**Pull request merged**